### ISSUE TITLE
Xterm: Can't execvp python: No such file or directory

### ISSUE THREAD

**Issue by tanazgh** [Screetsec/BrutSploit#43]
When I want to run some modules that open up other xterm terminal this error appears.
pls help!

**Comment by sinamotraghi** [comment]
i have the exact same problem.
any resolves?

**Comment by sinamotraghi** [comment]
when i set username and evrything for instagram attack and type run an other terminal pops up that says : Xterm: Can't execvp python: No such file or directory

**Comment by tanazgh** [comment]
yes. I actually solved it by switching to ubuntu instead of using wsl,
cause wsl does not support multi windows linux gui applications.
But if you already use ubuntu you can try installing "python" or "python3"
on your xterm terminal or try solutions from following links:

https://askubuntu.com/questions/419641/panel-apps-try-to-launch-in-xterm-even-though-it-isnt-installed
(changing x-terminal alternatives)
https://www.raspberrypi.org/forums/viewtopic.php?t=228156   (creating xterm
manually in usr/bin if it's not there)

I hope it helps.

**Comment by v33ga** [comment]
add path for python, you can check it with "echo $PATH"
